data_validation/config_manager.py

```diff
@@ -763,15 +800,15 @@ def append_pre_agg_calc_field(
                 self.source_client.name == "bigquery"
                 or self.target_client.name == "bigquery"
             ):
-                calc_func = "cast"
+                calc_func = consts.CONFIG_CAST
                 cast_type = "timestamp"
                 pre_calculated_config = self.build_and_append_pre_agg_calc_config(
                     source_column,
                     target_column,
                     calc_func,
                     column_position,
-                    cast_type,
-                    depth,
+                    cast_type=cast_type,
+                    depth=depth,
```

Comment:
Why these changes were needed? Not clear to me.

Reply:
Those arguments are named and not positional and therefore, in my opinion, calling code should reference them by name. The change was not "needed" but IMO that is how they should be used. Because I was referencing them in my other changes I also modified the call here.

Reply:
I had to search a little bit to confirm my initial thought and according to the link below it looks like if parameters are not declared in the function with the special characters that enforce positional-only or keyword-only behavior, either approach can be used with no problems. So it doesn't matter at the end as both ways work, anyway it was a good to recap this Python detail!

https://docs.python.org/3/tutorial/controlflow.html#positional-or-keyword-arguments